index.html

```diff
@@ -38,49 +38,72 @@ <h1>Queer Tech Bristol</h1>
-        <ul>
-          <li class="events__event">
-            <div class="events__description">
-              <a 
-                href="https://www.eventbrite.co.uk/e/queer-tech-bristol-show-and-share-feat-control-shift-tickets-450502895287"
-              >
-                <h3 id="event-2">Tuesday 15th November, 6-8pm</h3>
-              </a>
-              <p>
-                This month's QTB will be an opportunity to chat with our peers about our passion projects/side hustles/digital obsessions. Sign up to take part!
-              </p>
-              <a 
-                href="https://www.eventbrite.co.uk/e/queer-tech-bristol-show-and-share-feat-control-shift-tickets-450502895287"
-              >
-                get your free ticket here</a>
-            </div>
-            <div>
-              <img
-                class="events__image"
-                alt="a dithered close up image of a rose"
-                src="./images/showcase.gif"/>
-            </div>
-          </li>
-          <li class="events__event">
-            <div class="events__description">
-              <a 
-                href="https://www.eventbrite.co.uk/e/queer-tech-bristol-hacktoberfest-special-tickets-430408612747"
-              >
-                <h3 id="event-2">Tuesday 18th October, 6-8pm</h3>
-              </a>
-              <p>
-                We're back at the Watershed for our second session - a hack night to take part in <a href="https://hacktoberfest.com/">Hacktoberfest</a >.
-              </p>
-              <a 
-                href="https://www.eventbrite.co.uk/e/queer-tech-bristol-hacktoberfest-special-tickets-430408612747"
-              >
-                get your free ticket here</a>
-            </div>
+        <section class="upcoming-events">
+          <h2 id="upcoming-events">Upcoming events</h2>
+          <ul>
+            <li class="events__event">
+              <div class="events__description">
+                <a 
+                  href="https://www.eventbrite.co.uk/e/queer-tech-bristol-queermas-a-talk-about-giving-talks-tickets-475629339157"
+                >
+                  <h3 id="event-2">Tuesday 20th December, 6-8pm</h3>
+                </a>
+                <p>
+                  Our December event will feature a talk about giving talks and a seasonal social!
+                </p>
+                <a 
+                  href="https://www.eventbrite.co.uk/e/queer-tech-bristol-queermas-a-talk-about-giving-talks-tickets-475629339157"
+                >
+                  get your free ticket here</a>
+              </div>
+              <div>
+                <img
+                  class="events__image"
+                  alt="a dithered close up image of Cliff Richard riding a sled with his fist in the air"
+                  src="./images/queermas_cliff.png"/>
+              </div>
+            </li>
+          </ul>
+        </section>
+
+        <section class="past-events">
```

Comment:
it looks like the `past-events` and `current-events` classes aren't being used, did you mean to add some styling to them or did you mean them to be more semantic? If they are semantic or for accessibility, I think something like an `aria-label` or `aria-labelledby` tag makes more sense https://www.w3.org/WAI/GL/wiki/Using_HTML5_section_element 

If you *did* want to add styles, [please can we use BEM naming guidelines](https://getbem.com/introduction/) for css? so something like `events__past` maybe?